porting/debugging/hardware-debug/connectivity.rst

```diff
@@ -0,0 +1,48 @@
+Connectivity Debugging
+======================
+
+WiFi
+----
+Debug steps::
+
+    # Driver status
+    lsmod | grep wifi
+    dmesg | grep wifi
+    iw list
+    
+    # Connection
+    iw dev wlan0 scan
+    iw event
+    wpa_cli status
+
+Cellular
+--------
+Debug methods::
+
+    # Modem
+    mmcli -m 0
+    ofono-phonesim-cli
```

Comment:
No. Both of these commands are wrong.

- We don't use ModemManager.
- `ofono-phonesim` is essentially a SIM card simulator for oFono. We don't use that here; we have an actual modem.

Reply:
OK. Will remove.

Reply:
Done